infrastructure/environments/backend/pr.conf

```diff
@@ -0,0 +1,3 @@
+resource_group_name  = "byk-admin-rg"
```

Comment:
This is a pointless comment - but, there's project_name_short value - can we build up some of these names from these base variables? (don't do this in this PR - if ever...)

Reply:
Unfortunately, `project_name_short` comes later on in the terraform execution. This file is used when calling `terraform init`.